game/src/environment/towers/core/Tower.tscn

```diff
@@ -140,28 +137,18 @@ __meta__ = {
 [node name="CollisionShape2D" type="CollisionShape2D" parent="."]
 shape = SubResource( 1 )
 
-[node name="Range" type="Area2D" parent="."]
```

Comment:
Why the changes here? 143-149

Reply:
see https://github.com/bharathcs/quantum-defence/pull/61#pullrequestreview-699530283 and https://github.com/bharathcs/quantum-defence/pull/62